graylog2-web-interface/templates/index.html.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const vendorModule = JSON.parse(fs.readFileSync('build/vendor-module.json', 'utf8'));
+
+function linkTagForCssFile(cssFile) {
+  return '<link href="' + cssFile + '" rel="stylesheet">';
+}
+
+function scriptTagForJsFile(jsFile) {
+  return '<script src="' + jsFile + '"></script>';
+}
+
+module.exports = function (templateParams) {
+  const htmlWebpackPlugin = templateParams.htmlWebpackPlugin;
+  const manifest = htmlWebpackPlugin.files.manifest ? ' manifest="' + htmlWebpackPlugin.files.manifest + '"' : '';
+  const title = htmlWebpackPlugin.options.title ? htmlWebpackPlugin.options.title : 'Webpack App';
+  const favIcon = htmlWebpackPlugin.files.favicon ? '<link rel="shortcut icon" href="' + htmlWebpackPlugin.files.favicon + '">' : '';
+
+  const cssFiles = htmlWebpackPlugin.files.css.map(function(css) {
+    return linkTagForCssFile(htmlWebpackPlugin.files.css[css]);
+  }).join("\n");
+
+  const vendorFiles = Object.keys(vendorModule.files.chunks).map(function(chunk) {
+    return "\t" + scriptTagForJsFile('/' + vendorModule.files.chunks[chunk].entry);
+  });
+  const jsFiles = Object.keys(htmlWebpackPlugin.files.chunks).map(function(chunk) {
+    return "\t" + scriptTagForJsFile(htmlWebpackPlugin.files.chunks[chunk].entry);
+  }).join("\n");
+
+  return '\
+  <!DOCTYPE html>\n\
```

Comment:
Thanks for pushing me on to this again. I had a simple idea which helped me employing the old EJS template again. (See aecc82d)